server/build_without_gradle.sh

```diff
@@ -50,6 +50,7 @@ echo "Compiling java sources..."
 cd ../java
 javac -bootclasspath "$ANDROID_JAR" -cp "$CLASSES_DIR" -d "$CLASSES_DIR" \
     -source 1.8 -target 1.8 \
+    -encoding UTF-8 \
```

Comment:
`build_without_gradle.sh` would fail when being invoked from within a vanilla Ubuntu container. I think this is because `C_LANG` and friends were set to ASCII.  Since this repository uses UTF-8, it seemed to make sense to enforce that here as well, and this fixed the issue.
The PR as it currently stands no longer calls `build_without_gradle` directly so this is not needed for this MR.  I can undo the change if you want, but I don't think it's wrong ;-).